tests/generators_test.cpp

```diff
@@ -137,7 +137,6 @@ namespace generators
 
         PointSource source(freq, rayNumPerRow, 1);
```

Comment:
Note that test framework supports so called fixtures so that you do not have to create the same objects on every test.

https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests

Reply:
najs 👍